bin/cli.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+"use strict";
+
+const { Command } = require("commander");
+const { version } = require("../package.json");
+const { migrate, transform } = require("./commands");
+
+// Initial program setup
+const program = new Command();
+
+// `$ codemods version || codemods -v || codemods --version`
+program.version(version, "-v, --version", "Output the version number");
+program
+  .command("version")
+  .description("Output your version of Codemods")
+  .action(() => {
+    process.stdout.write(version + "\n");
+    process.exit(0);
+  });
+
+// `$ codemods generate`
+program
+  .command("migrate")
+  .description("Migrate Strapi applications from v3 to v4")
```

Comment:
I think we should specify a _single_ application

```suggestion
  .description("Migrate a v3 Strapi application or plugin to v4")
```

Reply:
Should be fixed in latest commit 🙌🏻